survey/sas-serial-preflight-plan.ps1

```
@@ -0,0 +1,385 @@
+        'ComputerSerial',
+        'Computer Serial',
+        'AssetSerial',
+        'Asset Serial',
+        'SN'
+    )
+}
+
+function Get-ProbeTargetFromRow {
+    param($Row)
+
+    $host = Get-RowValue -Row $Row -Names @('HostName', 'Hostname', 'ComputerName', 'DeviceName', 'Name', 'DnsName', 'DNSName', 'FQDN', 'IPAddress', 'IP', 'IPv4')
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'host' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)